examples/assembly_access-control-list.html

```diff
@@ -184,9 +188,11 @@ <h2 id="_additional_resources">Additional resources</h2>
 <a class="rh-docs-legal-notice__link" href="https://access.redhat.com/docs-legal-notice">Legal Notices for Trademarks</a>
 </div>
 </article>
+<!-- inject:footer:html -->
```

Comment:
Looks like the footer is inside of the cp-documentation tags, we should pull that out, into the doc